### ISSUE TITLE
Bulk tag items

### ISSUE THREAD

**Issue by VB6Enjoyer** [aardappel/procrastitracker#114]
I've been using the software and it has come in VERY useful, in fact, I really like it since it gives me better insight of my own behavior.

One issue I've been having, though, is that software like Firefox will normally have ProcrastiTracker track each tab individually, so right now, PT is tracking the time I'm spending on this tab, writing this, and then separately tracking the tab in which I'm watching an anime. These are two different activities, and I'd usually divide them between "web browsing" and "entertainment", separately, but it becomes quite complicated when you end up having hundreds of such tabs tracked, and you must tag each one individually.

It's currently impossible to select two items simultaneously, so you can only tag items one by one. Can it be made possible to select multiple things at once, or perhaps tag everything which passes a filter? It'd make everything MUCH easier.

**Comment by aardappel** [comment]
I agree that is sorely needed.. at the time I implemented it, I couldn't see how I could make multi-select work with the WIN32 api, so gave up :)
